API/API.mli

```diff
@@ -5666,7 +5666,7 @@ module Vernacinterp :
 sig
   type deprecation = bool
 
-  type vernac_command = Genarg.raw_generic_argument list -> unit -> unit
+  type vernac_command = Genarg.raw_generic_argument list -> Loc.t option -> unit
```

Comment:
In the current model (for better error reporting as far as I understood), the evaluation is done in two steps and this requires a partial application (see the call to `hunk`).